docs/workforce-management-framework/topics/con-demo-app.adoc

```diff
@@ -0,0 +1,6 @@
+[id='con-demo-app-{chapter}']
```

Comment:
Fair point - I'll add it to the assembly for now and if it grows, I can break it back out into a concept